mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -222,57 +223,63 @@ export const Columns = (props: ColumnsProps) => {
         };
 
         return (
-          <VerticalContainer>
+          <VerticalContainer className={styles.list}>
             <ol className={styles.numberedList}>
-              {row.original.jobsDone.map((jobDone) => (
+              {row.original.jobsDone.map((jobDone, index) => (
                 <li key={jobDone.uuid}>
-                  <HorizontalContainer>
-                    <HorizontalContainer className={styles.numberedListItem}>
+                  <HorizontalContainer className={styles.numberedListItem}>
+                    <VerticalContainer className={styles.verticalContainer}>
+                      <HorizontalContainer className={styles.horizontalContainer}>
```

Comment:
too much wrappers

Reply:
Deleted one horizontal wrapper but other refactoring let's move to another task please

Reply:
Task #371